validator/sawtooth_validator/journal/chain.py

```diff
@@ -359,19 +357,19 @@ def _find_common_height(self, new_chain, cur_chain):
             # new chain is longer
             # walk the current chain back until we find the block that is the
             # same height as the current chain.
-            while new_blkw.block_num > cur_blkw.block_num and \
-                    new_blkw.previous_block_id != NULL_BLOCK_IDENTIFIER:
+            while (new_blkw.block_num > cur_blkw.block_num
```

Comment:
Unless the linter is complaining, let's not make these sorts of stylistic changes